api/src/middleware/__tests__/competenciesRouter.ts

```diff
@@ -0,0 +1,54 @@
+import { collectionEnvelope } from '../responseEnvelope';
+import {
+  countCompetencies,
+  listCompetencies,
+} from '../../services/competenciesService';
+import { createAppAgentForRouter } from '../routerTestUtils';
+import competenciesRouter from '../competenciesRouter';
+
+jest.mock('../../services/competenciesService.ts');
+const mockCountCompetencies = jest.mocked(countCompetencies);
+const mockListCompetencies = jest.mocked(listCompetencies);
+
+describe('competenciesRouter', () => {
+  const appAgent = createAppAgentForRouter(competenciesRouter);
+
+  describe('GET /', () => {
```

Comment:
@BRashad The route in the Describe block is incorrect.  I believe it should say '/competencies', no?